filament/backend/src/vulkan/VulkanCommands.cpp

```diff
@@ -154,6 +154,17 @@ bool VulkanCommands::flush() {
     const int64_t index = mCurrent - &mStorage[0];
     VkSemaphore renderingFinished = mSubmissionSignals[index];
 
+    // VUID-vkEndCommandBuffer-commandBuffer-01815
+    // TODO: We need to order calls so that there is no outstanding label or markers.
+    if (!mCurrentDebugLabel.empty()) {
```

Comment:
```suggestion
    if (UTILS_UNLIKELY(!mCurrentDebugLabel.empty())) {
```

Reply:
looks like `UTILS_UNLIKELY` is a compiler hint for branching, but we're actually going to hit this branch on almost every frame. Still add it?

Reply:
Well, after understanding better that this is actually more a workaround, I think it doesn't matter. Eventually this code should go away (maybe an assert), once we figure out a better API for debug markers. So I'll leave up to you.